codespeed.conf

```diff
@@ -689,6 +723,16 @@ experiments:
                 suites:
                   - savina-jit
 
+    SOMns-SavinaCSP-exp:
+        description: Run the Savina Actor benchmarks
+        executions:
+            - SOMns-interp-tn-exp:
+                suites:
+                  - savina-csp
+            - SOMns-graal-tn-exp:
+                suites:
+                  - savina-csp
```

Comment:
We might need separate configs (suites) for the Graal and interpreter version.
I try to size them so that the Graal version runs 40-80ms in the optimized case.
The interpreter version should not run too long, we don't want to wait ages. :)

Reply:
separated suites and scaled benchmarks. interp still uses same problem size, difference in performance is small

Reply:
Hmm, perhaps that's the benchmarks, or my naive implementation of CSP...